### DISCUSSION TITLE
Remove cellColData info

### DISCUSSION THREAD

**Discussion by NoemieL** [GreenleafLab/ArchR#2128] [Questions / Documentation]
Hello,

I'd like to share my ArchR project with collaborators, but there's some information I'd like to delete.
How can I delete a special column from cellColData?

Thanks

**Comment by dgodovich** [comment]
Hey, 

I know this is super late but this came up when I was searching for how to do this and I just figured out how: you can use `proj@cellColData${column you want to remove} <- NULL` to remove the column.
